edb/schema/utils.py

```diff
@@ -320,21 +371,30 @@ def get_nq_name(schema, item) -> str:
 
 
 def find_item_suggestions(
-        name, modaliases, schema, *, item_types=(), limit=3,
-        collection=None, condition=None):
+        name: Union[sn.SchemaName, str, None],
```

Comment:
`sn.SchemaName` is a subclass of `str`, so I wonder if `Optional[str]` is enough here.

Reply:
Right. :+1: